pkgs/development/libraries/ffmpeg/generic.nix

```diff
@@ -212,9 +656,11 @@ stdenv.mkDerivation rec {
       No matter if they were designed by some standards committee, the community or
       a corporation.
     '';
-    license = licenses.gpl3;
-    maintainers = with maintainers; [ ];
+    license = with licenses; [ lgpl21Plus ]
+      ++ optional withGPL gpl2Plus
+      ++ optional withGPLv3 gpl3Plus
+      ++ optional withUnfree unfreeRedistributable;
     platforms = platforms.all;
-    inherit branch knownVulnerabilities;
+    maintainers = with maintainers; [ codyopel atemu ];
```

Comment:
```suggestion
    maintainers = with maintainers; [ atemu ];
```

Last interaction with this package back in 2015/05. I think it's safe to say they moved on.

Reply:
@codyopel